### PR TITLE
support only tensorflow v1.0

### PR DESCRIPTION
**Pull request by antlerros**

### CODE REVIEW

**Comment by thtrieu** [comment]
Commenting out the `'initializers'` part in 'net/ops/convolution.py' will prevent layer from being initialized by pretrained weights. Please fix this.

**Comment by antlerros** [comment]
initializer problem fixed, thank you.

**Comment by thtrieu** [comment]
`tf.concat` exists throughout 4 different scripts in this repo. Please add the version check to all of them (the `if v[0], v[1]` statement) and apply two versions of concat like in the `initializers` case. 

Only then, this commit will be fully *support tensorflow 1.0* :)

**Comment by antlerros** [comment]
fixed!

**Comment by thtrieu** [comment]
Well, maybe you did not see what I said in the previous comment:

> There  is `tf.concat` exists throughout 4 different scripts in this repo.

You fixed one. Three more to go :)

**Comment by antlerros** [comment]
hi, i fixed the four parts. sorry for the stupid miss read.

**Comment by thtrieu** [comment]
No problem @antlerros! I really appreciate the fact that you have the intention to spend time upgrading the project.

There is still something that I had to point out though: what I mean is `4 scripts`, not four places in a single script. The scripts that contains `tf.concat` are:

`net/yolov2/train.py, net/yolo/train.py, net/ops/convolution.py, net/ops/simple.py`

I have a suggestion to make your job easier: Let's move the whole project to tensorflow 1.0 and forget about older versions altogether. You can either do that, or follow the old way: adding a bunch of if statements to lots of places in these scripts.

Your choice :)

**Comment by antlerros** [comment]
ahh I see. I think my editor didn't find all of them (even though it states it searches through the whole project...)
yea i agree making it supports the new version of tensorflow only! 
Thank you :)

**Comment by antlerros** [comment]
Hi, it now supports only tensorflow v1.0
Please check, thanks! 😀

File: net/ops/convolution.py [hunk 1]
```diff
@@ -85,9 +86,9 @@ def batchnorm(self, layer, inp):
                 'updates_collections' : None,
                 'is_training': layer.h['is_training']
                 })
-            v = tf.__version__.split('.')[1]
-            if int(v) < 12: key = 'initializers'
-            else: key = 'param_initializers'
+
+            v = tf.__version__.split('.')
+            key = 'param_initializers'
```
Comment (thtrieu) [inline net/ops/convolution.py]: `v = tf.__version__.split('.')` is no longer needed.
`{'param_initializers' : layer.w}` should be put directly inside the first definition of `args` .

**Comment by antlerros** [comment]
Removed. Thanks.

**Pull request merged**